src/main/java/projet/projetstage02/controller/RootController.java

```diff
@@ -1036,4 +1037,34 @@ public ResponseEntity<InterviewOutDTO> StudentSelectDate(@RequestBody InterviewS
             return ResponseEntity.status(NOT_FOUND).build();
         }
     }
+
+    @PostMapping("/evaluateStudent/{token}")
+    public ResponseEntity<?> evaluateStudent(@PathVariable String token,
```

Comment:
Why is ResponseEntity return ?

Reply:
Because we dont return anything in the body. Since ResponseEntity is a generic class putting ? tells java that we don't know what it returns. Otherwise intellij throws warnings.